### ISSUE TITLE
Support cashaccount for `blockchain.address.*` calls

### ISSUE THREAD

**Issue by dagurval** [BitcoinUnlimited/ElectrsCash#123]
We already index cashaccounts, so might as well use our own index to support cashaccount in address calls.

**Comment by dagurval** [comment]
Ping @monsterbitar, I think you might like this idea

**Comment by monsterbitar** [comment]
yes, but how would it work out when there's multiple payment information? a subscription to a cashaccount subscribes to all of the payment info?

**Comment by dagurval** [comment]
I forgot that there could be multiple payment information. That makes it more difficult.

Subscribing to all would work. How about `get_balance`? Fetch the balance of all combined? And `blockchain.address.get_scripthash` would either need to error out or return a list of strings rather than a string.

**Comment by monsterbitar** [comment]
Perhaps it would be better to separate into new calls instead of overloading, since the end result is actually quite different. Particulary as cashaccounts are extensible and could support other payment information in the future that can't be directly compared with addressess.
